lib/components/tiled_component.dart

```diff
@@ -74,10 +76,84 @@ class TiledComponent extends Component {
       final dst = Rect.fromLTWH(tile.x.toDouble(), tile.y.toDouble(),
           rect.width.toDouble(), rect.height.toDouble());
 
-      c.drawImageRect(image, src, dst, paint);
+      //Rotating & Drawing our Tiles
+      switch(tile.rotation){
+        case 0:
+          //0 ° Rotation
+          c.drawImageRect(image, src, dst, paint);
+          break;
+        case 1:
+          //90 ° Rotation
+          c.save();
+          c.translate( dst.center.dx, dst.center.dy );
+          c.rotate((3 * math.pi/2));
+          c.translate( -dst.center.dx, -dst.center.dy );
+          c.drawImageRect(image, src, dst, paint);
+          c.restore();
```

Comment:
These lines are repeated on all the cases, the only thing that changes is the radians passed to the rotate method, you should encapsulate this in a method receiving the radian as a parameter, so we don't have duplicated code.